dev_tools/notebooks/notebook_test.py

```diff
@@ -87,7 +87,7 @@ def env_with_temporary_pip_target():
 
 
 @pytest.mark.slow
```

Comment:
Can we temporarily comment out this `slow` mark to include the test in this PR checks?

If the checks pass, let's put the mark back and merge.

Reply:
Done. Can you explain what you mean by check the CI here? I thought the checks were handling that, but I could be mistaken

Reply:
If test is decorated with `@pytest.mark.slow` it is not included in the CI checks executed for the PR and instead runs in scheduled [daily workflows](https://github.com/quantumlib/Cirq/actions/workflows/ci-daily.yml).  Removing/commenting the slow mark will run the test in PR checks and we'll know if the fix worked sooner.